docs/excel/excel-data-types-entity-card.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Excel JavaScript API data types entity value card
+description: Learn how to use entity value cards with data types in your Excel add-in.
+ms.date: 05/18/2022
+ms.topic: conceptual
+ms.prod: excel
+ms.localizationpriority: medium
+---
+
+# Use cards with entity value data types (preview)
+
+> [!NOTE]
+> Data types APIs are currently only available in public preview. Preview APIs are subject to change and are not intended for use in a production environment. We recommend that you try them out in test and development environments only. Do not use preview APIs in a production environment or within business-critical documents.
+>
+> To use preview APIs:
+>
+> - You must reference the **beta** library on the content delivery network (CDN) (https://appsforoffice.microsoft.com/lib/beta/hosted/office.js). The [type definition file](https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts) for TypeScript compilation and IntelliSense is found at the CDN and [DefinitelyTyped](https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-js-preview/index.d.ts). You can install these types with `npm install --save-dev @types/office-js-preview`. For additional information, see the [@microsoft/office-js](https://www.npmjs.com/package/@microsoft/office-js) NPM package readme.
+> - You may need to join the [Office Insider program](https://insider.office.com) for access to more recent Office builds.
+>
+> To try out data types in Office on Windows, you must have an Excel build number greater than or equal to 16.0.14626.10000. To try out data types in Office on Mac, you must have an Excel build number greater than or equal to 16.55.21102600.
```

Comment:
Consider moving this to an include file for this article and the other data types articles.